testing/py3-ansicolor/APKBUILD

```diff
@@ -0,0 +1,30 @@
+# Contributor: Keith Maxwell <keith.maxwell@gmail.com>
+# Maintainer: Keith Maxwell <keith.maxwell@gmail.com>
+pkgname=py3-ansicolor
+_pyname=ansicolor
+pkgver=0.2.6
```

Comment:
There's only 0.2.4 https://github.com/numerodix/ansicolor/releases

Reply:
otoh the proper source is https://pypi.org/project/ansicolor/#files

Reply:
I'm not clear what change you are suggesting, the logic for packaging as proposed is:

- Tests are not packaged on PyPI, I want to add the package with check() enabled
- 0.2.6 is released on PyPI but not tagged on GitHub ; 0.2.4 is five years old at this point
- The commit selected changes the release number to 0.2.6 <https://github.com/numerodix/ansicolor/commit/a5a5c31dc6de5c864a0c5684ae326972573a712b>

I have commented on the upstream issue asking the author to add a tag (or package tests on PyPI): <https://github.com/numerodix/ansicolor/issues/6>

Reply:
To avoid any potential delay, I will switch to 0.2.4